runtime/doc/cmdline.txt

```diff
@@ -41,15 +41,19 @@ thus you cannot edit beyond that.
 The command-lines that you enter are remembered in a history table.  You can
 recall them with the up and down cursor keys.  There are actually five
 history tables:
+
 - one for ':' commands
 - one for search strings
 - one for expressions
 - one for input lines, typed for the |input()| function.
 - one for debug mode commands
+
 These are completely separate.  Each history can only be accessed when
```

Comment:
This one is also necessary for the same reason. Because of the inconsistency of vimdoc, once a list item starts, it doesn't end until a blank line.